units/patch_intent.go

```diff
@@ -992,8 +992,18 @@ func (j *patchIntentProcessor) buildGithubPatchDoc(ctx context.Context, patchDoc
 		})
 	}
 
+	j.user, err = findEvergreenUserForPR(patchDoc.GithubPatchData.AuthorUID)
+	if err != nil {
+		return isMember, errors.Wrapf(err, "finding user associated with GitHub UID '%d'", patchDoc.GithubPatchData.AuthorUID)
+	}
+	patchDoc.Author = j.user.Id
+	patchDoc.Project = projectRef.Id
+
 	patchContent, summaries, err := thirdparty.GetGithubPullRequestDiff(ctx, githubOauthToken, patchDoc.GithubPatchData)
 	if err != nil {
+		if strings.Contains(err.Error(), thirdparty.PRDiffTooLargeErrorMessage) {
```

Comment:
Can you add a comment / a link to some github docs saying this is the expected response we get from Github when the diff is too large? It's not immeditately clear why we're checking this / what it means

Reply:
there's no documentation :( but i added a comment